src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/JsonSerializerDataContractResolver.cs

```diff
@@ -177,7 +184,7 @@ private IEnumerable<DataProperty> GetDataPropertiesFor(Type objectType, out Type
 
                     return
                         (property.IsPubliclyReadable() || property.IsPubliclyWritable()) &&
-                        !(property.GetIndexParameters().Any()) &&
+                        !(property.GetIndexParameters().Length > 0) &&
```

Comment:
@martincostello is this another performance-related suggestion? Personally, I prefer the readability of `Any()`

Reply:
Most likely yes. See [CA1860](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1860)

(Note: I also don't like that rule. I've disabled it in my projects since I don't write high performance code.)